formatter/json.go

```diff
@@ -76,7 +77,8 @@ func (f *Formatter) jsonPrint(issues tflint.Issues, tferr *tflint.Error) {
 
 	if tferr != nil {
 		var errs []error
-		if diags, ok := tferr.Cause.(hcl.Diagnostics); ok {
+		var diags *hcl.Diagnostics
```

Comment:
Why is this a pointer and not a value? From `errors.As`:

> An error matches target if the error's concrete value is assignable to the value pointed to by target

Reply:
Otherwise, you'll meet this error :
```
errorsas: second argument to errors.As must be a non-nil pointer to either a type that implements error, or to any interface type (govet)
```